internal/kubernetes/operator/datafederation/datafederation.go

```diff
@@ -54,8 +55,8 @@ func BuildAtlasDataFederation(dataFederationStore store.DataFederationStore, dat
 		},
 		Spec: getDataFederationSpec(dataFederation, targetNamespace, projectName),
 		Status: akov2status.DataFederationStatus{
-			Common: akov2status.Common{
-				Conditions: []akov2status.Condition{},
+			Common: api.Common{
```

Comment:
nit: let's stay consistent and use the alias `akoapi` (without the v2, as this is version-independent)